### PR TITLE
added GSA Cards/Tests

### PR DESCRIPTION
**Pull request by Achocholacek**

### CODE REVIEW

File: server/game/cards/03-WC/AEmbertracker.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const Card = require('../../Card.js');
+
+class AEmbertracker extends Card {
```
Comment (cryogen) [inline server/game/cards/03-WC/AEmbertracker.js]: We usually skip the E in cards that have "amber" in their name

**Comment by Achocholacek** [comment]
this is gonna require the updated wc.json to be merged before it'll work.

File: server/game/cards/03-WC/AEmbertracker.js (hunk 1 above)
Comment (Achocholacek) [reply server/game/cards/03-WC/AEmbertracker.js]: resolved. will require WC.json update as welll.

**Comment by cryogen** [comment]
the constant is added now

**Comment by romanogit** [comment]
Need a merge of this one :)

**Comment by cryogen** [comment]
It can't be merged until the issues are resolved with it

**Pull request merged**